### PR TITLE
feat: global find replace

### PR DESCRIPTION
**Pull request by mscolnick**
image

### CODE REVIEW

File: frontend/src/components/find-replace/find-replace.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        <Tooltip content="Find Next">
+          <Button size="xs" variant="secondary" onClick={() => findNext()}>
+            <ArrowLeftIcon className="w-4 h-4" />
+          </Button>
+        </Tooltip>
+        <Tooltip content="Find Previous">
+          <Button size="xs" variant="secondary" onClick={() => findPrev()}>
+            <ArrowRightIcon className="w-4 h-4" />
+          </Button>
+        </Tooltip>
```
Comment (akshayka) [inline frontend/src/components/find-replace/find-replace.tsx:150]: Let's switch these: `ArrowLeftIcon` should be `findPrev` and `ArrowRightIcon` should be `findNext`. I think that's more natural/standard.

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -273,7 +273,9 @@ export const App: React.FC<AppProps> = ({
   // HOTKEYS
   useHotkey("global.runStale", runStaleCells);
   useHotkey("global.save", saveOrNameNotebook);
-  useHotkey("global.interrupt", sendInterrupt);
+  useHotkey("global.interrupt", () => {
```
Comment (akshayka) [inline frontend/src/App.tsx:276]: Just curious, why make this change?

File: frontend/src/core/hotkeys/hotkeys.ts [hunk 3]
```diff
@@ -127,6 +127,11 @@ const DEFAULT_HOT_KEY = {
     group: "Editing",
     key: "Mod-f",
   },
+  "cell.selectNextOccurrence": {
+    name: "Select next occurrence",
+    group: "Editing",
+    key: "Mod-d",
```
Comment (akshayka) [inline frontend/src/core/hotkeys/hotkeys.ts:133]: For some reason, this doesn't work for me (it just bookmarks the page on Chrome, using Linux). Not a huge deal, just noting.

**Review by akshayka — COMMENTED** [review]
This is super nice! Elegant implementation too. Left a few small comments/questions. Main one about switching behavior of arrows. Otherwise looks great!

File: frontend/src/App.tsx (hunk 2 above)
Comment (mscolnick) [reply frontend/src/App.tsx:276]: `sendInterrupt` returns a Promise which messed with the typing changes I made to `useHotkey`

File: frontend/src/components/find-replace/find-replace.tsx (hunk 1 above)
Comment (mscolnick) [reply frontend/src/components/find-replace/find-replace.tsx:150]: oops, yea thats wrong

File: frontend/src/core/hotkeys/hotkeys.ts (hunk 3 above)
Comment (mscolnick) [reply frontend/src/core/hotkeys/hotkeys.ts:133]: this is my favorite hotkey in vscode. i was glad to see codemirror has it. did you have some text selected? and in an editor?

**Review by akshayka — APPROVED** [review]

**Pull request merged**